src/test/java/com/snowflake/kafka/connector/records/RecordContentTest.java

```diff
@@ -364,8 +368,48 @@ private void testGetProcessedRecordRunner(
     RecordService service = new RecordService();
     Map<String, Object> recordData = service.getProcessedRecordForStreamingIngest(record);
 
-    assert recordData.size() == 2;
+    assert recordData.size() == 4;
```

Comment:
is this size change due to 3 additional columns? if yes should not it be 5 (2+3 new)?